lib/formulator.ex

```diff
@@ -2,31 +2,85 @@ defmodule Formulator do
   use Phoenix.HTML
   alias Formulator.HtmlError
 
+  @doc  """
+  Returns an html input with an associated label.
+  If there are errors associated with the field, it will also output a span
+  tag with the errors.
+
+  ## Options
+
+    * `:label` - when false, does not create a label tag. Instead, an `aria-label`
+    attribute is added to the input to improve accessibility.
+    When given a string, that string is used as the label text.
```

Comment:
Can you make this the first sentence? I think this will be the more common use case.